### PR TITLE
Create infra-eng.md

### PR DESCRIPTION
**Pull request by djsoul6**
Would love your thoughts on this.

### CODE REVIEW

**Comment by djsoul6** [comment]
@drummel @jeremiemv @stuartparmenter in case you didn't get a notification to check this out.

File: infra-eng.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#this message is specific to building our early infrastrcuture team. for context, there is a follow-up email explaining more about what our engineering team does (aka build 1life).#
+
+Hi {{first_name}} -- Hope you're having a great {{day_of_week}}. I'm unsure what your situation is at {{company}} but we’re bolstering our Infrastructure/DevOps team at One Medical and would appreciate the opportunity to spread the gospel (so to speak).
+
+I wanted to reach out to you specifically because we’re looking for Senior/Staff Engineers who have been part of early infrastructure teams and played an integral role in design and implementation. From what I can tell you, that looks like you were apart of a lot of those decisions at [insert companies here]. I'm hoping you may be interested in using your skills to contribute to our mission to 'transform healthcare; in other words, help people like you and me manage their health and get great care when we need it.
```
[the remark this answers is not in the document]
Comment (drummel) [reply infra-eng.md]: "help people like you and me manage their health and get great care when we need it." this sounds convoluted... can't really parse this easily. If we say "in other words", it should be blunt and to the point. "keep people healthy"

... how about ...
"Our mission is to transform healthcare for people like you and me. We do this by building technology that lowers the barrier to getting care, improving both short and long term outcomes. In short, we keep people heathy.

File: infra-eng.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#this message is specific to building our early infrastrcuture team. for context, there is a follow-up email explaining more about what our engineering team does (aka build 1life).#
+
+Hi {{first_name}} -- Hope you're having a great {{day_of_week}}. I'm unsure what your situation is at {{company}} but we’re bolstering our Infrastructure/DevOps team at One Medical and would appreciate the opportunity to spread the gospel (so to speak).
```
[the remark this answers is not in the document]
Comment (djsoul6) [reply infra-eng.md]: good point!

File: infra-eng.md [hunk 3]
```diff
@@ -1,17 +1,14 @@
...
+I'm reaching out to you specifically because we’re looking for experienced engineers who played an integral role in designing and implementing early systems and/or redesigns. From what I can tell you, that looks like you were apart of a lot of those decisions at [insert companies here]. Our mission is to transform healthcare for people like you and me. We do this by building technology that lowers the barrier to getting care, improving both short and long term outcomes. In short, we keep people healthy.
 
 We spent a lot of time writing this job description and feel like it does a good job explaining the role: https://onemedical.com/jobs/details/?id=1100084&gh_jid=1100084
```
[the remark this answers is not in the document]
Comment (djsoul6) [reply infra-eng.md]: I think it's helpful to state that we actually spent time on it. Reason being is job descriptions often times aren't helpful bc they're generic boilerplate docs.

**Pull request merged**